src/components/pagination/pagination.config.js

```diff
@@ -79,16 +80,29 @@ const variants = [
     context: {
       version: 'v2',
       itemsPerPageChoices,
+      pageNumberChoices,
       totalPages: 5,
     },
   },
+  {
+    name: 'v2 Disabled Pagination Buttons',
+    label: 'V2 Disabled Pagination Buttons',
+    context: {
+      version: 'v2',
+      itemsPerPageChoices: [itemsPerPageChoices[0]],
+      totalPages: 1,
+      pageNumberChoices: [pageNumberChoices[0]],
+      disabledPaginationButton: true,
+    },
+    notes: `Notify the user of their position in the page range by disabling the appropriate pagination buttons at the start or end of the range.`,
+  },
 ];
 
 module.exports = {
   context: {
     featureFlags,
     prefix,
-    pageNumberChoices,
+    // pageNumberChoices,
```

Comment:
Are we removing this line...?

Reply:
yes I've removed it, left it in by accident